openpdf/src/main/java/com/lowagie/text/pdf/PdfBoolean.java

```diff
@@ -50,87 +50,43 @@
 
 package com.lowagie.text.pdf;
 
-import com.lowagie.text.error_messages.MessageLocalization;
-
 /**
- * <CODE>PdfBoolean</CODE> is the boolean object represented by the keywords <VAR>true</VAR> or <VAR>false</VAR>.
- * <P>
+ * Wrapper around {@code boolean} value.
+ *
  * This object is described in the 'Portable Document Format Reference Manual version 1.7'
  * section 3.2.1 (page 52).
  *
- * @see        PdfObject
- * @see        BadPdfFormatException
+ * @see PdfBoolean#TRUE
+ * @see PdfBoolean#FALSE
+ * @see PdfBoolean#fromValue(boolean)
  */
-
-public class PdfBoolean extends PdfObject {
-    
-    // static membervariables (possible values of a boolean object)
-    public static final PdfBoolean PDFTRUE = new PdfBoolean(true);
-    public static final PdfBoolean PDFFALSE = new PdfBoolean(false);
```

Comment:
It is probably a good Idea to leave these  constants there like
`public static final PdfBoolean PDFTRUE = TRUE;`
and deprecate them, so the change will not break existing user code.